doc/faq.rst

```diff
@@ -0,0 +1,229 @@
+    data = reg.default_data_container(64)
+
+If this fails, the pyrfr dependency is most likely not compiled correctly. We advice you to do the
+following:
+
+1. Check if you can use a pre-compiled version of the pyrfr to avoid compiling it yourself. We
+   provide pre-compiled versions of the pyrfr on `pypi <https://pypi.org/project/pyrfr/#files>`_.
+3. Check if the dependencies specified under :ref:`installation` are correctly installed,
+   especially that you have ``swig`` and a ``C++`` compiler.
+2. If you are not yet using Conda, consider using it; it simplifies installation of the correct
+   dependencies.
+4. Install correct build dependencies before installing the pyrfr, you can check the following
+   github issues for suggestions: `1025 <https://github.com/automl/auto-sklearn/issues/1025>`_,
+   `856 <https://github.com/automl/auto-sklearn/issues/856>`_
+
+Log files and output
+====================
+
+Where does Auto-sklearn output files by default?
+------------------------------------------------
+
+*Auto-sklearn* heavily uses the hard drive to store temporary data, models and log files which can
+be used to inspect the behavior of Auto-sklearn. Each run of Auto-sklearn requires
+its own directory. If not provided by the user, *Auto-sklearn* requests a temporary directory from
+Python, which by default is located under ``/tmp`` and starts with ``autosklearn_tmp_`` followed
+by a random string.
+
+The :class:`autosklearn.classification.AutoSklearnClassifier` and all other *auto-sklearn*
+estimators accept the argument ``tmp_directory`` which change where such output is written to.
+
+There's an additional argument ``output_directory`` which can be passed to *Auto-sklearn* and it
+controls where test predictions of the ensemble are stored if the test set is passed to ``fit()``.
+
+Auto-sklearn eats up all my disk space
+--------------------------------------
+
+*Auto-sklearn* heavily uses the hard drive to store temporary data, models and log files which can
+be used to inspect the behavior of Auto-sklearn. By default, *Auto-sklearn* stores 50
+models and their predictions on the validation data (which is a subset of the training data in
+case of holdout and the full training data in case of cross-validation) on the hard drive.
+Redundant models (i.e. when we have more than 50 models) are removed everytime the ensemble builder
+finishes an iteration, which means that the number of models stored on disk can temporarily be
+higher if a model is output while the ensemble builder is running. One can therefore change the
+number of models that will be stored on disk by passing an integer for the argument
+``max_models_on_disc``.
+
+As the number of models is only an indicator of the disk space used it is also possible to pass
+the memory in MB the models are allowed to use as a ``float``. As above this is rather a
+guideline on how much memory is used as redundant models are only removed from disk when the
```

Comment:
So what exactly does this limit? The memory per model, the memory used by all stored models (which is only ensured once the ensemble builder finished an iteration) or something else?